flink-runtime-web/src/main/resources/META-INF/NOTICE

```diff
@@ -1053,13 +1053,10 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 ---
 color-name@1.1.4
 ---
-The MIT License (MIT)
-Copyright (c) 2015 Dmitry Ivanov

```

Comment:
Can we remove a Copyright like this? copyright lines are normally something that would not be removed.

Reply:
Of course we can. I'm sorry, I wasn't paying attention. It's updated now, please take a look at it sometime, thanks!